src/cpp/cuda.hpp

```diff
@@ -997,8 +997,8 @@ namespace pycuda
       CUstream m_stream;
 
     public:
-      stream(unsigned int flags=0)
-      { CUDAPP_CALL_GUARDED(cuStreamCreate, (&m_stream, flags)); }
```

Comment:
Could you add a test?

Reply:
I did, but still struggling with the CI :/